src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project: Todo List</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="main-container">
+        <div>
+            <input type="text" value="Today's to do" id="task-input-1" class="left">
+        </div>
+        <div>
+            <input type="text" placeholder="Add to your list..." id="task-input-2" class="left">
+        </div>
+        <div id="todo-list"></div>
+        <div>
+            <a href="#" id="clear-completed" class="left">Clear all Completed</a>
+        </div>
+    </div>
```

Comment:
- [ ] Please style the to-do list app according to the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg)
I have attached an annotated diagram so that you can see where you need to make changes 😎 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/230749283-0d609b64-102c-4142-948e-0e1d2e8a8e6a.png)
### Guideline
![image](https://user-images.githubusercontent.com/46839548/230749352-2980d26a-e11b-480b-a2d0-3372ea6ba2e9.png)

Reply:
Hello @codepantha thanks a lot for your review. Let me  fix it then. Good catch.

Reply:
Done!
<img width="1438" alt="Screenshot 2023-04-08 at 8 57 57 PM" src="https://user-images.githubusercontent.com/88105477/230750331-3e26c8ee-c7e9-49e1-8469-aa0a50440643.png">